app/services/savings_pass_fail_service.rb

```diff
@@ -0,0 +1,67 @@
+class SavingsPassFailService
```

Comment:
I don't see anywhere the min and max thresholds being stored on the `Saving` object. It's done in the migration for the existing applications, but not for the new ones. Unless I missed something?